client/human/HumanClientApp.cpp

```diff
@@ -626,12 +625,12 @@ void HumanClientApp::MultiPlayerGame() {
     auto server_connect_wnd = GG::Wnd::Create<ServerConnectWnd>();
     server_connect_wnd->Run();
 
-    std::string server_name = server_connect_wnd->Result().second;
+    std::string server_dst = server_connect_wnd->GetResult().server_dst;
```

Comment:
I don't know what `dst` refers to.

Reply:
Server destination

Reply:
`dest` would be better than `dst`, which looks like an initialism.

Reply:
Changed.